discovery/gossiper.go

```diff
@@ -2035,6 +2071,10 @@ func (d *AuthenticatedGossiper) updateChannel(info *channeldb.ChannelEdgeInfo,
 		BaseFee:         uint32(edge.FeeBaseMSat),
 		FeeRate:         uint32(edge.FeeProportionalMillionths),
 	}
+	chanUpdate.Signature, err = lnwire.NewSigFromRawSignature(edge.SigBytes)
```

Comment:
do this before creating the `ChannelUpdate` maybe

Reply:
Why before the channel update? This just assigns it in-line.

Reply:
Just a style preference. I prefer to set all fields when creating the struct.